### DISCUSSION TITLE
How do I knwo which notes are beeing used ? (bot doesn't find some notes)

### DISCUSSION THREAD

**Discussion by ottantaventi** [brianpetro/smart-connect#61] [Help & Support]
When using Smart Connect - Notes, how do I know which notes the bot is using to answer my query ?

Because I see often that he pulls up only part of the relevant notes or even doesn't find a note

Today I asked him twice if he was able to access a specific note by giving him and he was unable to find it, despite answering that he was able to access my notes.
Only the 3rd time I asked he was able to access the note and then answer my query:

https://chatgpt.com/share/6751c90b-3854-800f-9d7d-0e385e877d98

I have an underlying mistrust that the system really searches all my notes, those my request about how can I now which notes he used to answer my query.

**Comment by brianpetro** [comment]
Check the console logs in the Smart Connect app. Every time ChatGPT interacts with your Smart Environment you can see ChatGPT's request and Smart Connect's response in the logs 🌴
